pom.xml

```diff
@@ -240,19 +240,17 @@ flexible messaging model and an intuitive client API.</description>
     <jclouds.version>2.6.0</jclouds.version>
     <guice.version>5.1.0</guice.version>
     <sqlite-jdbc.version>3.47.1.0</sqlite-jdbc.version>
-    <mysql-jdbc.version>8.0.11</mysql-jdbc.version>
-    <postgresql-jdbc.version>42.5.5</postgresql-jdbc.version>
+    <mysql-jdbc.version>9.1.0</mysql-jdbc.version>
```

Comment:
`mysql-jdbc.version` is completely unused in the Pulsar build. I don't see references anywhere. Please remove this completely.